src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java

```diff
@@ -30,6 +44,31 @@ public void playVideo() {
     lightboxVideo.click();
   }
 
+  public void verifyPlayerOnPage() {
+    wait.forElementPresent(By.id(ARTICLE_VIDEO_ID));
+  }
+
+  public void verifyPremiumPrerollRequest(NetworkTrafficInterceptor networkTrafficInterceptor, AdsOoyalaObject page) {
+    networkTrafficInterceptor.startIntercepting();
+    wait.forElementVisible(By.cssSelector(ARTICLE_VIDEO_WRAPPER_SELECTOR));
+    articleVideoClickArea.click();
+    page.wait.forSuccessfulResponseByUrlPattern(networkTrafficInterceptor, PATTERN_DFP_PREROLL);
+  }
+
+  public void verifyArticleAd() {
+    verifyColorAd(articleVideoWrapper, BLUE, AD_DURATION_SEC);
+    PageObjectLogging.log("ArticleAd",
+                          "Article had " + BLUE + " during " + AD_DURATION_SEC
+                          + " seconds", true);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal " seconds" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)